src/main/java/com/github/tomakehurst/wiremock/http/ResponseDefinition.java

```diff
@@ -334,12 +362,39 @@ public Fault getFault() {
     }
 
     @JsonInclude(NON_EMPTY)
-    public Parameters getTransformerParameters() {
-        return transformerParameters;
+    @JsonProperty("transformerInstances")
+    public List<String> getTransformerInstanceStrings() {
```

Comment:
I assume the JSON serialization is used for the `toString`, hence added this method. Please let me know if there is deserialization that can be negatively impacted by this new field. 

The reason to use `.toString` instead of exposing the transformers directly to JSON is that I prefer to play safe and avoid the situation where the custom transformation for whatever reason isn't nicely serializable, besides out of the box JSON serialization might not be very useful in many cases.